hedera-node/hedera-file-service-impl/src/main/java/com/hedera/node/app/service/file/impl/ReadableUpgradeStoreImpl.java

```diff
@@ -40,10 +43,12 @@
  * <p>This class is not exported from the module. It is an internal implementation detail.
  */
 public class ReadableUpgradeStoreImpl implements ReadableUpgradeStore {
```

Comment:
Same for this, why can't we use `ReadableFileStoreImpl` ?

Reply:
The File is in KV state, the content (the bytes of the file) are still filled in from the queue state. If it did not do that it would level the node.
This is still necessary to avoid if/else in every method of `ReadableFileStoreImpl`.